metron-platform/metron-common/src/test/java/org/apache/metron/common/utils/StellarExecutor.java

```diff
@@ -0,0 +1,84 @@
+package org.apache.metron.common.utils;
```

Comment:
Pushed the latest, which I think reflects your ideas @ottobackwards .  Take a look when you get a chance.